velocity-engine-core/src/main/java/org/apache/velocity/runtime/RuntimeInstance.java

```diff
@@ -405,9 +405,9 @@ private char getConfiguredCharacter(String configKey, char defaultChar)
         String configuredChar = getString(configKey);
         if (configuredChar != null)
         {
-            if (configuredChar.length() != 2 || configuredChar.getBytes(StandardCharsets.UTF_8).length != 1)
+            if (configuredChar.length() != 1)
             {
-                throw new RuntimeException(configKey + " must be a single byte UTF-8 character");
+                throw new RuntimeException(configKey + " must be a single UTF-8 character");
```

Comment:
`IllegalArgumentException` would be more suitable in this case. Moreover, the message is misleading. UTF-8 is not a character set, just an encoding and `configKey` is fine, but the value isn't. I'd prefer something like'

```
throw new IAE(String.format("value of '%s' must be a single character string, but is '%s'", configKey, configuredChar)
```

I think one can collapse both ifs into one.